Gruntfile.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function(grunt) {
+  'use strict';
+  grunt.loadNpmTasks('grunt-fh-build');
+  grunt.registerTask('default', ['eslint']);
```

Comment:
Yes, I would also say that we should not use grunt fh:build. It makes the tasks harder to read when we start adding testing tasks etc.

I'd say standard grunt components are better for open source.